sdk/core/core-rest-pipeline/src/constants.ts

```diff
@@ -1,6 +1,6 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-export const SDK_VERSION: string = "1.13.1";
+export const SDK_VERSION: string = "1.12.3";
```

Comment:
Is this a mistake?

Reply:
Fix. The change was on top of an old commit.